src/signals/incident/components/IncidentPreview/components/Image/index.js

```diff
@@ -20,7 +20,7 @@ const Image = ({ label, value }) => (
 
 Image.propTypes = {
   label: PropTypes.string,
-  value: PropTypes.string
+  value: PropTypes.array
```

Comment:
Shouldn't this be `PropTypes.arrayOf(PropTypes.string)`?

Reply:
gedaan